lib/datalog/roletester/src/role_tester.rs

```diff
@@ -105,10 +105,11 @@ fn create_error_ptr(e: String) -> Output {
 ///
 /// This function should not be called if input is invalid
 #[no_mangle]
-pub unsafe extern "C" fn process_access(input: *mut c_uchar, input_len: size_t) -> *mut Output {
+pub unsafe extern "C" fn process_access(input: *const c_uchar, input_len: size_t) -> *mut Output {
```

Comment:
I took the liberty of making this a const pointer - it does not seem like `input`is used mutably anywhere. `BytesMut::from` only accepts an immutable `&[u8]`, which it then internally clones into its own owned `Vec`.